### ISSUE TITLE
User-level error reporting is lacking

### ISSUE THREAD

**Issue by fedorov** [MGHComputationalPathology/slim#85]
When I get a 404 error, there is nothing that gets communicated to the user in the interface - only in the JS console. I think this should be improved. What do you think @hackermd @Punzo ?

![image](https://user-images.githubusercontent.com/313942/152852320-66c358dd-754e-4d75-b5b8-25259ca75d5d.png)

**Comment by hackermd** [comment]
Yes, that doesn't seem to be handled correctly and should be improved.

**Comment by fedorov** [comment]
This might have been addressed in https://github.com/ImagingDataCommons/slim/pull/134 - we need to confirm, in particular by testing against various problematic situations, such as 404.
